figures/scripts/supp-subtype-umap.R

```diff
@@ -0,0 +1,212 @@
+# Run Jin and Jo Lynne Rokita 
+#
+# Generate figures with UMAP results 
+
+### Load libraries
+library(ggplot2)
+library(tidyverse)
+
+### Define directory
+root_dir <- rprojroot::find_root(rprojroot::has_dir(".git"))
+data_dir <- file.path(root_dir, "data", "release-v21-20210820")
```

Comment:
I think we should be more careful with specific release tags here. Since `data/` should always contain symlinks to the current release, just having this should be fine:
```suggestion
data_dir <- file.path(root_dir, "data",)
```
Alternatively, if we want to be specific about a data release, `"release-v21-20210820"` could be passed in as an argument.

Reply:
@sjspielman - thanks for the suggestion. Modified accordingly.